docs/book/src/topics/using-iam-roles-in-mgmt-cluster.md

```diff
@@ -0,0 +1,44 @@
+# Using IAM roles in management cluster instead of AWS credentials
```

Comment:
These instructions could also apply to using IRSA with EKS with minimal changes. Shoudl we also cover this?

Reply:
Awesome @Ankitasw :tada:

Reply:
@richardcase I would be covering option 4 in this PR as described in [this](https://kubernetes.slack.com/archives/CD6U2V71N/p1648536004859049) slack thread. But you listed IRSA as option 2 in the thread, if it is possible with minimal changes I can do it in this PR itself.
Also, for enforcing use of IAM roles in EKS managed cluster, what additional step would be required apart from the steps listed in this doc?
cc @sedefsavas

Reply:
How about listing all options in that thread and If missing some details, could catch up during office hours and fill those.